uwblueprint/ccbc
# Omar/publishers bug fix

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Book-Publishers-disappear-on-edit-468aa9caa3f14c3f93860dfd7d25b26e)



## Implementation description
* 



## Steps to test
1. Create a Review with a book
2. Edit said review (change body, etc) or try editing the review's book in the frontend
3. Click Publish, go back to admin dashboard
4. Go the that review's edit page again and click on one of the books
5. Verify publishers appear correctly on the page



## What should reviewers focus on?
* 


## Checklist
